src/tests/cmake-build-debug/Project.cbp

```diff
@@ -0,0 +1,145 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CodeBlocks_project_file>
+	<FileVersion major="1" minor="6"/>
+	<Project>
+		<Option title="Project"/>
```

Comment:
What is this file? Accidental artifact?

Reply:
My bad. Committed accidentally with this file (generated by my editor).

Reply:
Thanks for reminding.